src/pages/Cart/component/CartBox.js

```diff
@@ -1,22 +1,116 @@
 import './CartBox.scss';
-const CartBox = () => {
+
+const CartBox = ({ product, getCart }) => {
+  const { image_url, name, quantity, product_id, id } = product;
+
+  const totalPrice = Number(product.price * quantity).toLocaleString('en');
+
+  const handleCountMinus = () => {
+    if (quantity <= 1) return;
+
+    fetch(`http://10.58.52.227:8000/carts/${id}`, {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+        Authorization: localStorage.getItem('token'),
+      },
+      body: JSON.stringify({
+        quantity: quantity - 1,
+      }),
+    })
+      .then(res => {
+        if (res.ok) {
+          getCart();
+        } else throw new Error('통신실패!');
+      })
+      .catch(error => console.log(error));
+  };
+
+  const handleCountUp = () => {
+    fetch(`http://10.58.52.227:8000/carts/${id}`, {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+        Authorization: localStorage.getItem('token'),
+      },
+      body: JSON.stringify({
+        quantity: quantity + 1,
+      }),
+    })
+      .then(res => {
+        if (res.ok) {
+          getCart();
+        } else throw new Error('통신실패!');
+      })
+      .catch(error => console.log(error));
+  };
+
+  const handleDelete = () => {
+    if (!window.confirm('정말 삭제하시겠습니까?')) return;
+
+    fetch(`http://10.58.52.227:8000/carts/${id}`, {
+      method: 'DELETE',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+        Authorization: localStorage.getItem('token'),
+      },
+    }).then(res => {
+      if (res.ok) {
+        getCart();
+      }
+    });
+  };
+
   return (
-    <section className="cart-box">
+    <section className="cart-box" key={product_id}>
       <div className="cart-img">
-        <img
-          src="/images/ProductDetail/plant-21.jpg"
-          alt="product-img"
-          className="img"
-        />
+        <img src={image_url} alt="product-img" className="img" />
       </div>
       <div className="cart-content">
         <div className="content">
-          <h1 className="title">몬스테라</h1>
-          <p>수량 : 2개</p>
-          <button className="btn">삭제</button>
+          <h1>{name}</h1>
+          <div className="count">
+            <button
+              className="btn btn-box"
+              onClick={() => {
+                handleCountMinus();
+              }}
```

Comment:
```suggestion
              onClick={handleCountMinus}
```
인자를 전달하지 않으면 이렇게 함수 이름으로 적용하시면 됩니다!
이 부분은 차후 다른 브랜치에서 함께 수정해주세요!

아래도 동일한 내용 있는거 전부 수정해주세요!